shinywidgets/_shinywidgets.py

```diff
@@ -110,7 +122,7 @@ def init_shiny_widget(w: Widget):
     # everything after this point should be done once per session
     if session in SESSIONS:
         return
-    SESSIONS.add(session)  # type: ignore
+    SESSIONS.add(session)
```

Comment:
Consider doing the session initialization before the widget initialization. And maybe in a separate function